src/cmdline/OptionParser.cxx

```diff
@@ -78,3 +78,46 @@ OptionParser::Next()
 
 	return {-1, nullptr};
 }
+
+const char *
+OptionParser::PeekOptionValue(const std::string &longOption)
```

Comment:
Bad variable name, and don't use `std::string` when `std::string_view` is enough, as your API would cause implicit copying to a `std::string` for no reason.